CHANGELOG.md

```diff
@@ -21,6 +21,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  - Improved the 2x2 downscaling method in the encoder for the optional color
    channel resampling for low bit rates.
 
+### Changed
+ - API: When encoding in lossy mode, the exponent_bits_per_sample in JxlBasicInfo
```

Comment:
this field was originally designed for lossy and represents only metadata about the original image. Is it maybe possible to only disable it in the situation where it causes issues?

Reply:
Since this PR changed completely, this question is no longer relevant :)